src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -734,4 +734,25 @@ They can be configured through the settings file 'winget settings'.</value>
   <data name="SourceOpenWithFailedUpdate" xml:space="preserve">
     <value>Failed in attempting to update the source:</value>
   </data>
+  <data name="UninstallCommandLongDescription" xml:space="preserve">
+    <value>Uninstalls the selected package, either found by searching the installed packages list or directly from a manifest. By default, the query must case-insensitively match the id, name, or moniker of the package. Other fields can be used by passing their appropriate option.</value>
+  </data>
+  <data name="UninstallCommandShortDescription" xml:space="preserve">
+    <value>Uninstalls the given package</value>
+  </data>
+  <data name="UninstallFlowStartingPackageUninstall" xml:space="preserve">
+    <value>Starting package uninstall...</value>
+  </data>
+  <data name="UninstallFlowUninstallSuccess" xml:space="preserve">
+    <value>Successfully uninstalled</value>
+  </data>
+  <data name="NoUninstallInfoFound" xml:space="preserve">
+    <value>The Windows Package Manager cannot locate the uninstall command for this application. Please reach out to the application publisher for support.</value>

```

Comment:
>The Windows Package Manager [](start = 11, length = 27)

nit: winget to be used in output message.
Also application should be replaced with package since we did a project wide renaming effort to use "package" to be consistent